plugins/experimental/parent_select/consistenthash.cc

```diff
@@ -248,11 +256,13 @@ PLNextHopConsistentHash::getHashKey(uint64_t sm_id, TSMBuffer reqp, TSMLoc url,
 }
 
 static void
-makeNextParentErr(const char **hostname, size_t *hostname_len, in_port_t *port)
+makeNextParentErr(const char **hostname, size_t *hostname_len, in_port_t *port, bool *retry, bool *no_cache)
 {
   *hostname     = nullptr;
   *hostname_len = 0;
   *port         = 0;
+  *retry        = false;
+  *no_cache     = false;
```

Comment:
This is how people who didn't go to graduate school would do this:  https://godbolt.org/z/M3z5fY46n